DOCS.md

```diff
@@ -0,0 +1,25 @@
+Use the Terraform plugin to apply the infrastructure configuration contained within the repository. The following parameters are used to configure this plugin:
+
+* `dryRun` - if true, calculates a plan but does __NOT__ apply it.
```

Comment:
Please use camelcase as we are doing it for all plugins so far. Difficult to change that later on...